lib/audio_service.dart

```diff
@@ -2242,7 +2242,23 @@ mixin QueueHandler on BaseAudioHandler {
 enum AudioServiceShuffleMode { none, all, group }
 
 /// The available repeat modes.
-enum AudioServiceRepeatMode { none, one, all, group }
+///
+/// This defines how media items should repeat when the current one is finished.
+enum AudioServiceRepeatMode { 
+  /// When the current media item finishes, playback will stop.
+  none, 
```

Comment:
This won't necessarily stop after the current media item finishes. For example, if there is a next item in the queue.

Reply:
Ah, right! Sorry, I don't know how to edit my pull request.

Reply:
If you just push additional commits to your branch, they should show up automatically in your pull request.